docs/integrations/app-development/opentelemetry/jira.md

```diff
@@ -0,0 +1,225 @@
+
+## Fields creation in Sumo Logic for Jira
+
+Following are the tags which will be created as part of the Jira App install if not already present. 
+
+`sumo.datasource` - Has fixed value of **jira**.
+
+## Prerequisites
+
+This section provides instructions for configuring log collection for Jira running on a non-Kubernetes environment for the Sumo Logic App for Jira. Sumo Logic supports the collection of logs from Jira server in standalone environments.
+
+Follow the instructions to set up log collection.
+
+The Jira app uses the following log types:
+-   **Jira Access Logs**. HTTP server access log files, in `logs/access_logs`,  in your JIRA Installation Directory.
+-   **Jira Security Logs**.Security-related information, such as logins, logouts, session creation/destruction, and security denials from `atlassian-jira-security.log`.
+-   **Jira Catalina Logs**. The application catalina log file, `logs/catalina.out`, in your JIRA Installation Directory.
+-   **Jira Webhooks**. Webhook events of types:
+    -   Issue
+    -   User
+    -   Sprint
+
+## Configure hosted collector to receive Webhooks
+
+In this step, you create a host collector to receive webhooks from Jira, and set up an HTTP source on it.
+
+1.  Configure a [hosted collector](https://help.sumologic.com/docs/send-data/hosted-collectors/configure-hosted-collector/), or select an existing hosted collector for the HTTP source.
+2.  Configure an [HTTP source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/) on the hosted collector.
+    -   For Source Category, specify Atlassian/Jira/Events
+    -   Add `sumo.datasource` as File to this source and assign the value **jira** for it.
+    <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Jira-OpenTelemetry/Jira-Datasource.png')} alt="Datasource" />
+    -   Make a note of the HTTP address for the source. You will supply it when you configure a Jira webhook in the next step.
+
+## Register webhook in Jira
+
+
+Follow the instructions on [Webhooks](https://developer.atlassian.com/server/jira/platform/webhooks/) in Jira help to register a webhook for the following events:
+
+-   Issue related:
+    -   created (jira:issue_created)
+    -   updated (jira:issue_updated)
+    -   deleted (jira:issue_deleted)
+    -   worklog changed (jira:worklog_updated)
+-   User related:
+    -   created (user_created)
+    -   updated (user_updated)
+    -   deleted (user_deleted)
+-   Sprint related:
+    -   created (sprint_created)
+    -   deleted (sprint_deleted)
+    -   updated (sprint_updated)
+    -   started (sprint_started)
+    -   closed (sprint_closed)
+
+When you configure the webhook, enter the URL for the HTTP source you created in [step 2](https://help.sumologic.com/docs/integrations/app-development/jira/#Step_2:_Configure_hosted_collector_to_receive_webhooks) as the endpoint for the webhook.
```

Comment:
```suggestion
When you configure the webhook, enter the URL for the [HTTP source you created](https://help.sumologic.com/docs/integrations/app-development/jira/#configure-hosted-collector-to-receive-webhooks) as the endpoint for the webhook.
```

Reply:
Hi @jpipkin1 , since we use relative links in docs, this would be `/docs/integrations/app-development/jira/#configure-hosted-collector-to-receive-webhooks`